File: src/main/java/br/com/blz/testjava/controller/dto/WarehouseDTO.java
package br.com.blz.testjava.controller.dto;

import br.com.blz.testjava.model.WarehouseType;

public class WarehouseDTO {
    private String locality;
    private int quantity;
    private WarehouseType type;

    public String getLocality() {
        return locality;
    }

    public void setLocality(String locality) {
        this.locality = locality;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }

    public WarehouseType getType() {
        return type;
    }

    public void setType(WarehouseType type) {
        this.type = type;
    }
}

File: src/main/kotlin/br/com/blz/testjava/controller/dto/WarehouseDTO.kt
package br.com.blz.testjava.controller.dto

import br.com.blz.testjava.domain.entities.Warehouse
import br.com.blz.testjava.domain.entities.WarehouseType

class WarehouseDTO() {
  var locality: String = ""
  var quantity: Int = 0
  var type: WarehouseType = WarehouseType.PHYSICAL_STORE

  constructor(warehouse: Warehouse): this() {
    this.locality = warehouse.locality
    this.quantity = warehouse.quantity
    this.type = warehouse.type
  }
}
